web/src/app/users/UserContactMethodForm.js

```diff
@@ -11,6 +11,8 @@ export default class UserContactMethodForm extends React.PureComponent {
       name: p.string.isRequired,
       type: p.oneOf(['SMS', 'VOICE']).isRequired,
       value: p.string.isRequired,
+      // config value for disclaimer is used if navigated here from first login attempt
```

Comment:
Let's keep this general (e.g. "disclaimer text to display at the bottom of the form") so it's more clear what it does, rather than where we expect it to be used.